domain/schema/model/sql/0024-relation.sql

```diff
@@ -155,7 +155,8 @@ INSERT INTO relation_status_type VALUES
 (1, 'joined'),
 (2, 'broken'),
 (3, 'suspending'),
-(4, 'suspended');
+(4, 'suspended'),
```

Comment:
todo: Please verify this list with `core/status/status.go` and `core/relation/status.go`. Looks like there may be duplicates.

Reply:
The relations status types are a bit of a mess. The one in `core/status/status.go` includes everything except error, the one in `core/relation.status.go` doesn't include any "ing" ones but does include error, and theres also `rpc/params/status.go` which includes only `joined`, `suspended` or `broken`.

All of them have the base type string and are converted from one to another, and since Go does not restrict the types in any way, these could actually be anything.

I added the `error` status because it is explicitly set to that by the firewaller, so we know it is actually used. The rest are all used somewhere, but its hard to tell if they are really needed.

It might be good to loop back to this once we have things working and tidy up the different types a bit.